From: Jens Axboe <axboe@suse.de>
Subject: [PATCH] fix via82xx resume

Trying software suspend on my workstation makes it crash on resume. The
problem is that via82xx marks the chip_init function as _devinit, but
calls it on resume as well.

Signed-off-by: Jens Axboe <axboe@suse.de>

--- a/sound/pci/via82xx.c	2005-04-30 03:30:54.000000000 +0200
+++ b/sound/pci/via82xx.c	2005-05-03 13:31:12.000000000 +0200
@@ -1781,7 +1781,7 @@
  *
  */
 
-static int __devinit snd_via82xx_chip_init(via82xx_t *chip)
+static int snd_via82xx_chip_init(via82xx_t *chip)
 {
 	ac97_t ac97;
 	unsigned int val;
